aries-core/src/stories/navigation/content/AppShell.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useContext } from 'react';
+import { Box, Grid, Main, ResponsiveContext, type BoxProps } from 'grommet';
+import { AppHeader } from './AppHeader';
+import { ContextPane } from './ContextPane';
+
+const gridAreas = [
+  ['nav', 'header', 'context-pane'],
+  ['nav', 'main', 'context-pane'],
+];
+
+const responsiveGridAreas: Record<string, string[][]> = {
+  xsmall: [
+    ['nav', 'header', 'context-pane'],
+    ['main', 'main', 'context-pane'],
+  ],
+};
+
+interface AppShellProps {
+  navigationMenu: React.ReactNode;
+  mainContent: React.ReactNode;
+  contextContent?: string;
+  setContextContent: (value: string) => void;
+  setActiveItem: (value: string) => void;
+}
+
+export const AppShell = ({
+  navigationMenu,
+  mainContent,
+  contextContent,
+  setContextContent,
+  setActiveItem,
+}: AppShellProps) => {
+  const breakpoint = useContext(ResponsiveContext);
+
+  // Only round the top-right corner of the main content when the context pane is visible
+  const mainRound: BoxProps['round'] = contextContent
+    ? { corner: 'top', size: 'small' }
```

Comment:
I just want to verify should the left side also be `rounded` when the nav is closed. 

<img width="369" height="221" alt="Screenshot 2025-12-16 at 3 13 12 PM" src="https://github.com/user-attachments/assets/b9206a24-dd25-4c1c-8374-3f07bbf26e0e" />